modules/apps/analytics/analytics-client-js/src/utils/map.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+function convertMapToArr(mapInstance) {
+	const arr = [];
+
+	mapInstance.forEach((value, key) => {
+		arr.push([key, value]);
+	});
+
+	return arr;
+}
+
+function getMapKeys(mapInstance) {
+	const arr = [];
+
+	mapInstance.forEach((value, key) => {
+		arr.push(key);
```

Comment:
Could we remove the curly brackets here and maybe have just one line?

Reply:
We can't because `.map` isn't available fot `Map` objects